Python-path-traversal-files-main/Kirubanantham7/Download-Center/m.py

```
@@ -0,0 +1,259 @@
+@app.route('/download_file/<path:video_filename>')
+def download_file(video_filename):
+    try:
+        decoded_filename = unquote(video_filename)
+        safe_name = secure_filename(decoded_filename)
+        file_path = os.path.join(get_download_folder(), safe_name)
+        return send_file(file_path, as_attachment=True, download_name=safe_name)
+    except Exception as e:
+        return f"An error occurred: {str(e)}", 500
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Ruff (0.13.3)

Use of `debug=True` in Flask app detected

(S201)